README.md

```diff
@@ -1,43 +1,43 @@
-## CloudFoundry PHP Example Application: Drupal 
+## cloud.gov example application: Drupal 7
```

Comment:
if we do choose to keep them in the same repo, we will probably have to create two top level sub-directories (e.g. `druapl7` and `drupal8`) so that you could cd into either then push